client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -59,7 +59,16 @@ const appShape = PropTypes.shape({
   appListing: PropTypes.arrayOf(PropTypes.shape({}))
 });
 
-export const AppPlaceholder = ({ apps }) => {
+export const AppPlaceholder = ({ apps, error }) => {
+  if (error.isError)
+    return (
+      <div
```

Comment:
I think we'll want to use `<SectionMessage type="warning">` here like in https://github.com/TACC/Core-Portal/blob/d136b2a9b892e68ca91db2d9bfb88a753beab26d/client/src/components/Applications/AppForm/AppForm.js#L99